src/main/java/edu/kis/powp/jobs2d/transformations/changedriver/ApplyTransformationToNewDriver.java

```diff
@@ -22,11 +20,9 @@ public static void applyTransformation(Job2dDriver transformation) {
         DrawerFeature.getDrawerController().clearPanel();
         LinesRecorder.getLinesRecorder().clearLines();
 
-        ICommandManager manager = CommandsFeature.getCommandManager();
-        manager.setCurrentCommand(RecordCommandsForTransformation.getRecordedCommand());
+        CommandsFeature.getCommandManager().setCurrentCommand(RecordCommandsForTransformation.getRecordedCommand());
 
-        DriverCommand command = CommandsFeature.getCommandManager().getCurrentCommand();
-        command.execute(DriverFeature.getDriverManager().getCurrentDriver());
+        CommandsFeature.getCommandManager().getCurrentCommand().execute(DriverFeature.getDriverManager().getCurrentDriver());
```

Comment:
No. You still use CommandManager. Executing commands do not require commands manager. You can simply do:
RecordCommandsForTransformation.getRecordedCommand().execute(DriverFeature.getDriverManager().getCurrentDriver())).